script.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
```

Comment:
what is use of use strict ?

Reply:
It defines that JavaScript should be executed in "strict mode".
For example, with strict mode we cannot use undeclared variables.